---
layout: misc
title: Syllabus
---

The syllabus is open to discussion with the instructor but the default set of topics is listed below.
After discussion, we can modify specific lectures to meet the needs
of indivdiuals in the class.

<hr />

<h4><span style="color: #0000ff;"><span class="c6 c11">Module 1 – Advanced R Programming.</span></span></h4>
<hr />

 
<p class="c2">Lecture 1: Programming revisited
Goal: To be ready to program.
Topics include: installing the programming language R and associated interactive development environment (IDE) RStudio. Also Python. </p>
 
<p class="c2">Lecture 2: R revisited.
Goal: A brief review of the basic in R. </p>
 
<p class="c2">Lecture 3-5: Advanced R.
Goal: Introduction to advanced programming practices in R. Shiny, ggplot2, plyr, plyr2, objects, general large scale software design.</p>

 <hr />

 <h4><span style="color: #0000ff;"><span class="c6 c11">Module 2 – Data Science of Next Generation Sequencing.</span></span></h4>
<hr />

<p class="c2">Lecture 6: Reproducible Science & Jupytr.
Goal: Introduction to a package that allows users to interactively describe statistical analyses and datasets in R and Python.</p>
 
<p class="c2">Lecture 7: Cloud Computing with R. 
Goal: To understand how to use cloud-based tools for large-scale computing (eg Google Cloud, Compute Canada, Azure, etc.</p>
 
<p class="c2">Lecture 8: Visualization in R and Python. 
Goal: To learn how to design and implement visualizations of data and statistical analyses.</p>

<p class="c2">Lecture 9-10: Basics of processing next generation sequencing data.
Goal: Develop skills using Python and R to manipulate next generation sequencing data including data preprocessing, quality control, and pipeline construction.</p>
 
<p class="c2">Lectures 11: Normalization of next generation sequencing data.  
Goal: Understand the concept of normalizing RNA-seq data. Contrast and compare different normalization protocols.</p>
 
<p class="c2">Lecture 12-13: Visualization of (single cell) RNA-seq data. 
Topics building upon previous experience with heatmaps but extend to t-SNE, diffusion maps, and derivatives.</p>
 
<hr />

 <h4><span style="color: #0000ff;"><span class="c6 c11">Module 3 – Novel Analytic Approaches for Single Cell Data.</span></span></h4>
<hr />

<p class="c2">Lecture 14: Single cell experimental design.  </p>
 
<p class="c2">Lecture 15: Handling missing data in single cell experiments.  </p>
 
<p class="c2">Lecture 16-18: RNA velocity, cell state and cell hierarchies.</p>
 
<p class="c2">Lecture 19-21: Inferring regulatory networks from single cell data. 
Topics include PERTURB-seq (MIMOSCA), SINCERITIES, and other inference approaches. Deep learning approaches </p>

