---
layout: blog
title: Single cell -omics
author: Mike Hallett
---

The wet lab portion of our group develops single cell profiling assays 
across  different modalities and in different contexts.
Our goal is not necessarily to advance genomics; rather we are focused on generating the right kind of data with good experimental design and an affordable cost.
Although we do work with commerical systems including, for example, the Chromium system (10X Genomics Inc), 
we are also interested in engineering these systems ourselves.
Although our lab does not offer a service per se, 
we  collaborate with many  groups in a research context with
our assays described belief below.


1. Transcriptional profiling via sc-DROP-seq as per [Macosko et al](http://dx.doi.org/10.1016/j.cell.2015.05.002)

2. Transcriptional profiling via various modified sc-DROP-seqs inspired by [Stephenson et al.](https://www.nature.com/articles/s41467-017-02659-x) and [Booeshaghi et al.](https://www.biorxiv.org/content/10.1101/521096v1) for self-design and printing of components.

3. We recently developed a fungal DROP-seq [bioRxiv](https://www.biorxiv.org/content/10.1101/2020.01.21.914549v1) The main differences between fungi and mammalian cells is that fungi have a cell wall that needs to be removed before they are lysed, and they have considerably less mRNA; with [Samira Massahi](https://www.mikehallett.science/team/samira-massahi/) and
[Van Bettauer](https://www.mikehallett.science/team/van-bettauer/).

4. DART-seq [Saikia et al.](https://doi.org/10.1038/s41592-018-0259-9) is a modified DROP-seq protocol that allows for simultaneous multiplexed amplicon sequencing and transcriptome profiling of single cells.

5. DRONC-seq [Habib et al.](https://www.nature.com/articles/nmeth.4407) is a DROP-seq based method to sequecing individual nuclei; we have been working to get this running in FFPE tumor material;
[Samantha Yuen](https://www.mikehallett.science/team/sam-yuen/).


6. sc-snare-seq2 [Chen et al.](https://doi.org/10.1038/s41587-019-0290-0), a DROP-seq approach that profiles both the transcriptome and chromatin accessibility of individual cells;
with [Vanessa Dumeaux](https://lab-dumeaux.science/) and [Suhani Patel](https://www.mikehallett.science/team/suhani-patel/).

7. We are in the process of setting up several types of sci-SPLIT-seq 
[Cao et al](DOI: 10.1126/science.aam8940) and PETRI-seq [Blattman et al](http://dx.doi.org/10.1101/866244) approaches based on 
barcoding of cells without microfluidics;
with [Brandon Finlay](http://www.findlaylab.ca/), [Shawn Simpson](https://www.mikehallett.science/team/shawn-simpson/).

8. We are developing PERTURB-seq in mammalian systems. This work
expands on many papers including the seminal manuscripts from 
[Adamson et al](doi:10.1016/j.cell.2016.11.048) and [Dixit et al](http://dx.doi.org/10.1016/j.cell.2016.11.038).

All of this work is supported by  bioinformatic pipelines for the basic processing and quality control of the resultant data. Our primary interest as a group however is in developing the data science expertise for the deep analysis of data towards   hypotheses specific to each project. 
This includes methods from [statsitics, computational, data science, and deep learning](https://www.mikehallett.science/research/deep-learning/).



This work has been developed with different members of our [team](https://www.mikehallett.science/team/) 
but very often in collaboration with [Vanessa Dumeaux](https://lab-dumeaux.science/) at PERFORM, Concordia,
and a broad range of additional collaborators.


<img class="pull-left" height="35" width="50" src="/images/cihr_logo.jpg">
<img class="pull-left" height="35" width="50"  src="/images/iric.png">
<img class="pull-left" height="100" width="100" src="/images/perform.gif">
<img class="pull-left" height="100" width="100"    src="/images/nserc.jpg">
<img class="pull-left" height="100" width="100" src="/images/crc.png">
<img class="pull-left" height="100" width="100" src="/images/Innovation_Logo.png">
<img class="pull-left" height="100" width="100" src="/images/concordia.logo.big.png">




