---
layout: blog
title: Biological networks
author: Mike Hallett
---

Our lab has several projects related to the inference of biological networks.
We have developed approaches that perturb specific sets of genes or
transcripts using genome editing (eg CRISPR) in a high-throughput,
multivariate manner. The edits can be knockouts, knock-downs or over-expression of the targets, or a combination thereof.
Single cell high-throughput profiling of transcription, select proteins, or chromosomal conformations are used to deduce underlying biological networks - the regulatory and correlative relationships between genes and gene products.

These projects bring together all the components of our lab:
<ul>
<li> Single cell multi-modal -omic profiling; </li>
<li> Genome editing; </li>
<li> Next generation sequencing; </li>
<li> Data science and analysis of high-throughput data; and </li>
<li> Deep learning for inference of biological networks. </li>
</ul>

[Sanny Khurdia](https://www.mikehallett.science/team/sanny-khurdia/), [Aki Kirbizakis](https://www.mikehallett.science/team/eftyhios-kirbizakis/) 
and [Van Bettauer](https://www.mikehallett.science/team/van-bettauer/) (computational biology) are leads on these projects.
We are also lucky to have [Aaybod](https://www.mikehallett.science/team/abdelrahman-ahmed/) assisting  as part of his genomics diploma.



We apply this to different biological systems including in the context of breast cancer with [Sylvie Mader's group](https://www.iric.ca/en/research/principal-investigators/sylvie-mader) at the IRIC/UdeM. 

<img class="pull-left" src="/images/cihr_logo.jpg">
<img class="pull-center" src="/images/iric.png">
<img class="pull-right" src="/images/nserc.jpg">

<img class="pull-left" src="/images/crc.png">
<img class="pull-left" src="/images/Innovation_Logo.png">
<img class="pull-left" src="/images/concordia.logo.big.png">
