---
layout: blog
title: Single cell -omics
author: Mike Hallett
---

We have been actively developing single cell profiling assays 
across  different modalities and in different contexts.
Although we do work with commerical systems including, for example, the Chromium system (10X Genomics Inc), 
we are also interested in engineering these systems ourselves.
Although our lab does not offer a service per se, 
we  collaborate with many  groups in a research context with
our assays:

1. Transcriptional profiling via sc-DROP-seq as per Macosko et al. (2015) Cell.

2.Ttranscriptional profiling via various modified sc-DROP-seqs inspired by [Stephenson et al.](https://www.nature.com/articles/s41467-017-02659-x) and [Booeshaghi et al.](https://www.biorxiv.org/content/10.1101/521096v1) for self-design and printing of components.

3.We recently developed a fungal DROP-seq. The main differences between fungi and mammalian cells is that fungi have a cell wall that needs to be removed before they are lysed, and they have considerably less mRNA; with [Samira Massahi](https://www.mikehallett.science/team/samira-massahi/).

4.DRONC-seq, a DROP-seq based method to sequecing individual nuclei; we have been working to get this running in FFPE tumor material;
[Samantha Yuen](https://www.mikehallett.science/team/sam-yuen/)


5. sc-snare-seq2, a DROP-seq approach that profiles both the transcriptome and chromatin accessibility of individual cells;
with [Vanessa Dumeaux](https://lab-dumeaux.science/) and [Suhani Patel](https://www.mikehallett.science/team/suhani-patel/).

6. We are in the process of setting up several types of sc-SPLIT-seq, approaches based on barcoding of cells without microfluidics;
with [Brandon Finlay](http://www.findlaylab.ca/), [Shawn Simpson](https://www.mikehallett.science/team/shawn-simpson/).

All of this work is supported by  bioinformatic pipelines for the basic processing and quality control of the resultant data. Our primary interest as a group however is in developing the data science expertise for the deep analysis of this work. 
This includes methods from [statsitics, computational, data science, and deep learning](https://www.mikehallett.science/research/deep-learning/).


<img class="pull-left" height="100" width="100" src="/images/cihr_logo.jpg">
<img class="pull-center" height="100" width="100"  src="/images/iric.png">
<img class="pull-center" height="100" width="100"src="/images/perform.gif">
<img class="pull-right" height="100" width="100" src="/images/nserc.jpg">
<img class="pull-left" height="100" width="100" src="/images/crc.png">
<img class="pull-left" height="100" width="100" src="/images/Innovation_Logo.png">
<img class="pull-left" height="100" width="100" src="/images/concordia.logo.big.png">





This work has been developed with different members of our [team](https://www.mikehallett.science/team/) 
but very often in collaboration with [Vanessa Dumeaux](https://lab-dumeaux.science/) at PERFORM, Concordia,
and a broad range of additional collaborators.


<img class="pull-left" src="/images/nserc.png">
<img class="pull-left" src="/images/crc.png">
<img class="pull-left" src="/images/Innovation_Logo.png">
<img class="pull-left" src="/images/concordia.logo.big.png">
