---
layout: page
research: true
title: "Research interests of our group"
author: Mike Hallett
category: misc
---

{% include JB/setup %}

## What the lab is like

We maintain both wet (benchwork &#x1F97D; 	&#x1F97C; &#x1F9EB;) and dry (computational 	
&#128187;) labs, providing a very inter-disciplinary environment for our group members. Everyone is  encouraged  to do (at least a little bit of) both, and we are all  committed to helping new members with this cross-training through both formal and _ad hoc_  events.

We are committed &#x1F4AF;	to a diverse, equitable and inclusive environment where everyone can function at their absolute best &#127752; &#x1F3F3;&#xFE0F;&#x200D;&#x26A7;&#xFE0F; . Please see our [lab ethos](/ethos/) for a more detailed description of what this means for us.

Much of our lab work is based on modern "-omics"  	&#129516; &#x1F9EC; technologies. Currently, we develop
single cell profiling technologies and lentiviral-based strategies for high-throughput CRISPR/Cas9-mediated perturbation delivery. __So people with backgrounds in the life sciences and who enjoy lab work have opportunities with us.__

The lab has significant expertise in computational biology and data science. We are particularly interested in the concept of __expression signatures__ and __expression networks__ (you can learn a bit about them below). Ultimately we are interested in applying these techniques to the data from our perturbation studies  to build computational models that help us reason about  cellular causal and correlative relationships. __So people with quantitative backgrounds (computer science, mathematics, statistics) may enjoy developing probabilistic models and deep learning toolkits with us to tackle these problems.__

The lab also co-leads efforts to profile large breast cancer cohorts. In particular, we are interested in early "pre-invasive" forms of breast cancer including _ductal carcinoma in situ_. The lab has a long-standing interest in  _breast cancer informatics_ where we apply our novel assays and models to better understand the disease at the molecular level, and where we build expression signatures that may have clinical utility in the treatment of breast disease. __So people who are interested in clinical applications and translational research have opportunities with us.__

We are  a highly collaborative lab, participating in projects with different groups and  institutions. [Dr. Vanessa Dumeaux](https://lab-dumeaux.science/) is one such long-term collaborator (she's also my wife), and our labs share space and equipment; we often have joint lab meetings. The lab also organizes a yearly meeting in [systems biology](/barbados), giving us an excellent opportunity to interact with some of the best researchers in the world. We really love  Do-It-Yourself (DIY) science, and are committed to Open Access Science.

You can learn more about our group, how it works and its dynamics here in this presentation. Afterwards, we provide slightly more technical sketches of the major directions in the lab. We note however that we are always open  to good ideas from new members!

<p align="center"><iframe src="https://docs.google.com/presentation/d/1bJfzJurxiTOmOke-KjzgeK6Ph1b2ogB4kmeRKwtG44c/edit?usp=sharing" frameborder="0" width="1100" height="900" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe></p>





<br><br>

# Development of new deep learning techniques in the life sciences

From a methods perspective, our primary interest is in the development of  data science and machine learning techniques for the  analysis of  molecular data. 
We have  a number of on-going computational projects in the lab, many of which are  rooted in fundamental data science methodology (statistics, visualization, hypothesis testing) or deep learning. Most of these projects are motivated by, and support, the applications described below. Mike was a professor in the [School for Computer Science at McGill](https://www.cs.mcgill.ca) for ~20 years and has experience supervising students from the quantitative sciences.

<strong>There are MSc and PhD positions available for individuals with strong backgrounds in the quantitative sciences to work on methods building related to the projects described below.</strong>

<div class="row">
  <div class="column">
    <img src="/assets/images/nserc.jpg" alt="NSERC" style="width:5%">
  </div>
  <div class="column">
    <img src="/assets/images/Innovation_Logo.png" alt="CFI" style="width:5%">
  </div>
  <div class="column">
    <img src="/assets/images/crc.png" alt="CRC" style="width:5%">
  </div>
  <div class="column">
    <img src="/assets/images/western_longWhite.png" alt="Western" style="width:5%">
  </div>
</div>


<br><br>

# Systematically perturbing biological systems

Our lab has several projects related to the inference of biological networks.
We have developed approaches that perturb specific sets of genes or
transcripts using genome editing (eg CRISPR) in a high-throughput,
multivariate pseudo-random manner. The edits can be knockouts, knock-downs or over-expression of the targets, or a combination thereof.
Single cell high-throughput profiling of transcripts, select proteins, or chromosomal conformations are used to deduce underlying biological networks - the regulatory and correlative relationships between genes and gene products.

These projects bring together all the components of our lab:
<ul>
<li> Single cell multi-modal -omic profiling; </li>
<li> Genome editing; </li>
<li> Next generation sequencing; </li>
<li> Data science and analysis of high-throughput data; and </li>
<li> Deep learning for inference of biological networks. </li>
</ul>

We apply this to different biological systems including in the context of breast cancer with [Sylvie Mader's group](https://www.iric.ca/en/research/principal-investigators/sylvie-mader) at the IRIC/UdeM in the past. 

<strong>There are two graduate level positions (MSc, PhD) open.  There are opportunities for people with strong mathematics, computational and statistical background. The work  is primarily based on probabilistic modelling and deep learning focused on inferring biological networks from the data genearted by our assays. There are also opportunities for people with strong bench skills that would like to work with lentivirus-delivered CRISPR-mediated molecular perturbations. Please [reach out](/join.html/) if you are interested.</strong>

<div class="row">
  <div class="column">
    <img src="/assets/images/nserc.jpg" alt="NSERC" style="width:5%">
  </div>
  <div class="column">
    <img src="/assets/images/cihr_logo.jpg" alt="CIHR" style="width:5%">
  </div>
  <div class="column">
    <img src="/assets/images/Innovation_Logo.png" alt="CFI" style="width:5%">
  </div>
  <div class="column">
    <img src="/assets/images/crc.png" alt="CRC" style="width:5%">
  </div>
  <div class="column">
    <img src="/assets/images/western_longWhite.png" alt="Western" style="width:5%">
  </div>
</div>


<br><br>



# Identifying effective treatments for women with early breast disease

This is a [CIHR](http://www.cihr-irsc.gc.ca/e/193.html) funded project that is a collaboration with [Dr. Eileen Rakovitch](https://sunnybrook.ca/research/team/member.asp?t=12&page=529&m=137) at the [Sunnybrook Hospital](https://sunnybrook.ca/) in Toronto, Canada. 


[Ductal in situ carcinoma (DCIS)(https://www.mayoclinic.org/diseases-conditions/dcis/symptoms-causes/syc-20371889) is a very common form of breast lesion. 
DCIS is a non-obligate precursor to invasive "life-threatening" breast cancer 
invasive ductal carcinoma ([IDC](https://www.breastcancer.org/symptoms/types/idc)).
Specifically, not all DCIS will become invasive, if left untreated (at least not within the natural lifetime
of an individual). 
In constrast to indolent DCIS, these lesions will progress, escaping from the mammary duct, into the surrounding breast tissue. Eventually,  if left untreated, they [metastases](https://en.wikipedia.org/wiki/Metastasis) to other organs and tissues.
A woman with an indolent DCIS might decide in consultation with her health care practioners tol follow a milder form of therapy involving only breast conserving surgery ([BCS](https://en.wikipedia.org/wiki/Breast-conserving_surgery))
However, if there is evidence she has a more agressive DCIS, then treatment might include BCS with additional [radiotherapy](https://www.mayoclinic.org/tests-procedures/radiation-therapy/about/pac-20385162).
The fundamental problem is that we have no way currently to decide whether a woman with DCIS benefits from the additional radiotherapy: we do not have biomarkers to differentiate those DCIS likely to remain indolent from those that will progress to an invasive state.
Therefor,e current [clinical practice](http://www.cancer.ca/en/cancer-information/cancer-type/breast/treatment/radiation-therapy/?region=on) is cautious and leans towards the inclusion of radiotherapy.


Dr. Rakovitch is a radiotherapist who has had great foresight and energy to build a cohort [tumor banking](https://en.wikipedia.org/wiki/Tumor_Bank) of  DCIS cases in Ontario from 1993 until present. This cohort is very well annotated with patient, clinical and pathological information about the lesion and patient. We are interested in identifying a molecular signature that can predict whether a woman with DCIS would benefit from a regime with only BCS versus a harsher regime of BCS + raditiotherapy.
To do this, we are profiling Dr. Rakovitch's Ontario cohorts using next generation sequencing and analysis methods from computational biology.
This data and analyses allow us to glimpse how each such lesion evolved, the collection of somatic DNA events in each tumor and the changes in the regulation and expression of specific pathways. 
From this data, we are developing molecular signatures that have the ability to predict at time of diagnosis the benefit from the inclusion of radiotherapy to assist clinical decision making.

<strong>
We have positions open at all levels from undergraduate projects, graduate students, postdocs and research associates. Please [reach out](/join.html/) if you are interested.
</strong>



<div class="row">
  <div class="column">
    <img src="/assets/images/cihr_logo.jpg" alt="CIHR" style="width:5%">
  </div>
  <div class="column">
    <img src="/assets/images/logo-sb-hsc.png" alt="Sunnybrook" style="width:5%">
  </div>
  <div class="column">
    <img src="/assets/images/western_longWhite.png" alt="Western" style="width:5%">
  </div>
</div>

<br> <br>

# Phenotypic heterogeneity in the fungus <italic>Candida albicans</italic>

We recently modified the DROP-seq  protocol (a single cell sequencing method) from Macosko et. al. to function with the fungus _C. albicans_. The main differences between fungi and mammalian cells, where DROP-seq is well establsihed, are that the former have a cell wall that needs to be removed before lysis, and the latter have considerably more mRNA. Our system was used to profile  _C. albicans_ grown in the presence of different antifungal drugs. We observed heterogeneity in their response, with evidence of distinct subpopulations of survivors with differential survival responses
[bioRxiv preprint](https://www.biorxiv.org/content/10.1101/2020.01.21.914549v1). There is  many opportunties for the development of single-cell informatic techniques optimized for the fungal setting for students with quantitative backgrounds. In addition to the single cell profiling techniques, we have also been building deep learning tools for  fungal microscopy [Candescence](https://osf.io/qdxbp/). This is ongoing work.


This is joint work with 
PhD candidate [Samira Massahi](https://www.mikehallett.science/team/samira-massahi/),
[Vanessa Dumeaux](https://lab-dumeaux.science/) at Western, 
[Judy Berman](https://www.jbermanlab.com/), and 
[Malcolm Whiteway](https://sites.google.com/site/whitewaylab/home) at Concordia. 


<div class="row">
  <div class="column">
    <img src="/assets/images/nserc.jpg" alt="NSERC" style="width:5%">
  </div>
  <div class="column">
    <img src="/assets/images/cihr_logo.jpg" alt="CIHR" style="width:5%">
  </div>
  <div class="column">
    <img src="/assets/images/crc.png" alt="CRC" style="width:5%">
  </div>
  <div class="column">
    <img src="/assets/images/western_longWhite.png" alt="Western" style="width:5%">
  </div>
</div>


<br><br>

#  Collaborative efforts related to breast cancer

We have many more projects related to breast cancer that are highly inter-disciplinary in nature. Our collaborators span across Canada from Dalhousie to UBC, to the United States including MD Anderson, NYU and Alabama, and in Europe with researchers in Norway, Sweden, Holland and Italy. 
__We are always looking for individuals of all backgrounds that are intrested in cancer biology and would like to contribute to the data science underlying these studies.__


<div class="row">
  <div class="column">
    <img src="/assets/images/research/funding/banner-nihlogo.png" alt="NIH" style="width:20%">
  </div>
 
   <div class="column">
    <img src="/assets/images/research/funding/ERC.jpg" alt="ERC" style="width:20%">
  </div>
 
  <div class="column">
    <img src="/assets/images/cihr_logo.jpg" alt="CIHR" style="width:5%">
  </div>
 
  <div class="column">
    <img src="/assets/images/western_longWhite.png" alt="Western" style="width:5%">
  </div>
</div>

<br><br>




